Swifty360Player/Swifty360PlayerScene.swift

```diff
@@ -43,6 +43,7 @@ open class Swifty360PlayerScene: SCNScene {
         super.init()
         self.videoPlaybackIsPaused = true
         self.player = player
+        self.player.automaticallyWaitsToMinimizeStalling = false
```

Comment:
Same here we don't need that line as well, you have already set on init of `Swifty360View.swift` and `Swifty360ViewController.swift` which is already enough.